great-idea-website/index.html

```diff
@@ -15,49 +15,95 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+   
+    <header>
+      <nav class="nav-bar">
+          <a href= "#">Services</a>
+          <a href= "#">Product</a>
+          <a href= "#">Vision</a>
+          <a href= "#">Features</a>
+          <a href= "#">About</a>
+          <a href= "#">Contact</a>
+          </nav>
+          
+          <img src= "img/logo.png" alt="Great Idea! Company logo.">
+        
+        </header> 
+        <article>
+          <h1 class="intro-title"> 
+          INNOVATION </br>
+             ON </br>
+             DEMAND</br>
+         <button>Get Started</button>
+         </h1>
+  </div>
+  <div class="header-right">
+    <img class="header-right-image">
+    <img src="img/header-img.png" alt="Image of a code snippet">
+    </article>
```

Comment:
You could increase the margin bottom for this article block, it's very close to the next block.